examples/BasicOperations/GetExpandedTextAds.php

```diff
@@ -121,14 +122,11 @@ public static function runExample(GoogleAdsClient $googleAdsClient, $customerId,
         foreach ($response->iterateAllElements() as $googleAdsRow) {
             /** @var GoogleAdsRow $googleAdsRow */
             $ad = $googleAdsRow->getAdGroupAd()->getAd();
-            // Note that the status printed below are enum values.
-            // For example, a value of 2 will be returned when the status is 'ENABLED'.
-            // A mapping of enum names to values can be found at AdGroupAdStatus.php
             printf(
-                "Expanded text ad with ID %d, status %d, and headline '%s - %s' was found in ad "
+                "Expanded text ad with ID %d, status '%s', and headline '%s - %s' was found in ad "
                 . "group with ID %d.%s",
                 $ad->getId()->getValue(),
-                $googleAdsRow->getAdGroupAd()->getStatus(),
+                AdGroupAdStatus::name($googleAdsRow->getAdGroupAd()->getStatus()),
                 $ad->getExpandedTextAd()->getHeadlinePart1()->getValue(),
```

Comment:
getHeadlinePart1Value. Please do this for all the files.

Reply:
Done